README.md

```diff
@@ -36,22 +36,31 @@ jobs:
         with:
           token: ${{ github.token }}
           branch: main
+          major-list: break
+          minor-list: feat, feature
+          patch-list: fix, bugfix, perf, refactor, test, tests
 
       - name: Create Release
         uses: ncipollo/release-action@v1
         with:
           allowUpdates: true
           draft: false
           name: ${{ steps.semver.outputs.next }}
+          tag: ${{ steps.semver.outputs.next }}
           body: Changelog Contents
           token: ${{ github.token }}
 ```
 
 ## Inputs
 * `token`: Your GitHub token (e.g. `${{ github.token }}`) - **REQUIRED**
 * `branch`: The branch to use when fetching list of commits to compare against. (e.g. `main`) - **Optional**
+* `major-list`: Comma separated commit prefixes, used to bump Major version. Defaults to empty - **Optional**
```

Comment:
Should mention that a `BREAKING CHANGE` note in the commit message will still trigger a major bump:

```markdown
* `major-list`: Comma separated commit prefixes, used to bump Major version. Defaults to empty. A `BREAKING CHANGE` note in a commit message will still cause a major bump. - **Optional**
```